collect_app/src/main/java/org/odk/collect/android/widgets/ItemsetWidget.java

```diff
@@ -75,15 +79,29 @@ public ItemsetWidget(Context context,
         this.fileUtil = fileUtil;
 
         items = getItems();
+    }
+
+    @Override
+    public IAnswerData getAnswer() {
+        Selection selectedItem =  ((SelectOneListAdapter) recyclerViewAdapter).getSelectedItem();
```

Comment:
Could you use `getSelectedItems().get(0)` instead to avoid having to cast? Generally having to cast is dangerous because it can lead to runtime errors. This same pattern shows up in a couple other places.

Reply:
Maybe not directly getSelectedItems().get(0) because it might be empty but yeah something like:
```
recyclerViewAdapter.getSelectedItems().isEmpty() ? null : recyclerViewAdapter.getSelectedItems().get(0);
```
will be possible.

Reply:
Fixed in https://github.com/getodk/collect/pull/3964/commits/e757ea8213db99fe18d88133e6bda0687b232878